index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <p>Hello Microverse</p>
```

Comment:
- [ ] Please use a h1 tag based on the [requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements)

Reply:
Thankyou @prolajumokeoni for your review. I have made the proposed changes as summarized below;

1. Added test.md and set it to be ignored by git.
2. Used h1 tag as per the requirements.